setup.py

```diff
@@ -1,8 +1,33 @@
 from setuptools import setup
 
+
+"""
+This code is from the python documentation and is
+designed to read in the version number.
+See: https://packaging.python.org/en/latest/guides/single-sourcing-package-version/
+"""
+import codecs
+import os.path
```

Comment:
This goes back to a discussion @w-k-jones and me had on using `pathlib` instead of `os.path` (#113 ). One could also remove the `codecs` import, but then this would not be compatible with Python 2 anymore I guess. If we wanna be more Python 3, it could look like:
```

from pathlib import Path

def read(pkg_name):
    init_fname = Path(__file__).parent / pkg_name / '__init__.py'
    with open(init_fname, "r") as fp:
        return fp.read()

def get_version(pkg_name):
    for line in read(pkg_name).splitlines():
        if line.startswith("__version__"):
            delim = '"' if '"' in line else "'"
            return line.split(delim)[1]
    else:
        raise RuntimeError("Unable to find version string.")

PACKAGE_NAME = "tobac"

setup(
    name=PACKAGE_NAME,
    version=get_version(PACKAGE_NAME),
```